### PR TITLE
ci: added github actions workflow

### PR DESCRIPTION
**Pull request by audrium**
Here is an example of GH actions workflow run: https://github.com/audrium/reana-message-broker/actions/runs/343641446

### CODE REVIEW

**Review by ParthS007 — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Python dependencies
+        run: |
+          pip install --upgrade pip setuptools py
+          pip install twine wheel
+          pip install -e .[all]
+
+      - name: Run pytest
```
Comment (mvidalgarcia) [inline .github/workflows/ci.yml:120]:
```suggestion
      - name: Run isort
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        run: ./run-tests.sh --check-isort
```
Comment (mvidalgarcia) [inline .github/workflows/ci.yml:121]: https://github.com/audrium/reana-message-broker/runs/1342402433?check_suite_focus=true#step:5:7

```
Run ./run-tests.sh --check-isort
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/isort/main.py:995: UserWarning: W0502: The following deprecated single dash CLI flags were used and translated: -df!
  warn(
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/isort/main.py:1000: UserWarning: W0501: The following deprecated CLI flags were used and ignored: -rc!
  warn(
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/isort/main.py:1004: UserWarning: W0500: Please see the 5.0.0 Upgrade guide: https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/
  warn(
```

Following [the migration guide](https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/) I think it should be `isort -c --df -- */*.py`

This probably applies to many other repos.

Comment (audrium) [reply .github/workflows/ci.yml]: After offline discussion it was decided to remove `isort` and use `black` instead

**Review by mvidalgarcia — APPROVED** [review]

**Comment by mvidalgarcia** [comment]
Cannot merge due to lack of privileges:

```console
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: At least 1 approving review is required by reviewers with write access.
To github.com:reanahub/reana-message-broker
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:reanahub/reana-message-broker'
```

**Pull request merged**